back/engines/commercial/analytics/app/models/analytics/dimension_user.rb

```diff
@@ -7,6 +7,7 @@
 #  id            :uuid             primary key
 #  role          :text
 #  invite_status :string
+#  is_visitor    :boolean
```

Comment:
Added this so that we can query participants and filter for the ones that accepted cookies